### DISCUSSION TITLE
Does sandbox has a git url

### DISCUSSION THREAD

**Discussion by CITGuru** [codesandbox/codesandbox-client#1913] [General]
# ❔ Question

I am trying to work on a new feature on codesandbox and I am wondering if there's a git repo url for each sandboxes.

**Comment by lbogdan** [comment]
Hey @CITGuru ,

There's not one at the moment, can you tell us why do you need it?

**Comment by CITGuru** [comment]
I am working on deploying sandbox to heroku. Theres already a deploy to heroku button integration that only requires you to provide the repo url and some app schema through the url params.

**Comment by CITGuru** [comment]
@lbogdan sandboxes are exportable to github. Is there a way to check if a sandbox is exported and also get the repo url

**Comment by blowsie** [comment]
For me the export function does not work, it just gets stuck
![image](https://user-images.githubusercontent.com/308572/62526524-52964980-b831-11e9-8355-657c5532eb7c.png)
